src/filters/pages.cjs

```diff
@@ -319,6 +319,14 @@ const byYear = (collection) => {
     }));
 };
 
+const addCallToAction = (pageURL) => {
+  const ctaPages = ['work', 'services'];
+  return ctaPages.some((page) => pageURL.toString().includes(page));
```

Comment:
I'm worried this is too permissive. If we write any blog post that contains `work` or `services` in the file name, this will return `true` and add the CTA.

Reply:
Actually this is already true on e.g. https://deploy-preview-264--oddleventy.netlify.app/tags/django-rest-framework/